docs/DebuggerHTML-Components-Reducers-Actions.md

```diff
@@ -0,0 +1,663 @@
+Debugger.html
+=============
+
+Debugger.html is an open source project that is built on top of React and Redux that functions as a standalone debugger for Firefox, Chrome and Node.  The project is being tested currently in Firefox Nightly and is available in the DevTools debugger tab when the preference devtools.debugger.new-debugger-frontend is enabled. This project is being created to provide a debugger that is stand-alone and does not require a specific browser tool to do debugging.
+
+This document gives a detailed view of the components, actions and reducers that make up the debugger.html project. Prior knowledge of React and Redux  is suggested.

React documentation can be found [here](https://facebook.github.io/react/docs/getting-started.html).
Redux documentation can be found [here](http://redux.js.org/).

As with most documentation related to code, this document may be out of date. The last edit date occurred on August 30, 2016.
+
+#Architecture
+
+
+Debugger.html is a React-Redux based application — the UI is constructed using React Components. the follow illustration provides a simplictic high level view:

```

Comment:
It might be nice to have a TOC

* Architecture
* Components
  * Editor
  * File Search
* Reducers